packages/pigment-css-react/src/Stack.d.ts

```diff
@@ -0,0 +1,20 @@
+import * as CSS from 'csstype';
+
+import { Breakpoint } from './base';
+import { PolymorphicComponent } from './Box';
+
+type CssProperty<T> = T | Array<T> | Partial<Record<Breakpoint, T>>;
+
+type StackBaseProps = {
+  display?: CssProperty<'flex' | 'inline-flex'>;
+  spacing?: CssProperty<number | string>;
+  direction?: CssProperty<CSS.StandardLonghandProperties['flexDirection']>;
+  justifyContent?: CssProperty<CSS.StandardLonghandProperties['justifyContent']>;
+  alignItems?: CssProperty<CSS.StandardLonghandProperties['alignItems']>;
```

Comment:
We should remove `display`, `justifyContent`, and `alignItems`

Reply:
Done.